### PR TITLE
Add alpha features in job pull-kubernetes-e2e-gce-alpha-features

### LINKED ISSUE #23642: Remove non-alpha features from job ci-kubernetes-e2e-gci-gce-alpha-features
**What should be cleaned up or changed**:
Jobs [`ci-kubernetes-e2e-gci-gce-alpha-features`](https://github.com/kubernetes/test-infra/blob/c22abc29319ca84054c05b326abf41a722c10d49/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L584) and [`pull-kubernetes-e2e-gce-alpha-features`](https://github.com/kubernetes/test-infra/blob/c22abc29319ca84054c05b326abf41a722c10d49/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L375) run for several features which are no longer in alpha status. Cleaning up the features requires:
1. Updating the job configurations to no longer focus non-alpha features.
2. (Optional, up to discussion) Adding new alpha features which should be tested by these jobs.
3. Ensuring the job is running properly. 

**Provide any links for context**:
Here are all the features tested by both jobs, together with its status and KEP link:

Feature | Status | KEP Link
----|----|----
ServiceAccountIssuerDiscovery | Stable | https://github.com/kubernetes/enhancements/issues/1393
StorageVersionAPI | **Alpha** | https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/2330-migrating-api-objects-to-latest-storage-version 
Audit | Stable | https://github.com/kubernetes/enhancements/issues/22
PodPreset | **Alpha** | https://github.com/kubernetes/enhancements/issues/162
RunAsGroup | Stable | https://github.com/kubernetes/enhancements/issues/213
TTLAfterFinished | Stable | https://github.com/kubernetes/enhancements/issues/592
NodeLease | Stable | https://github.com/kubernetes/enhancements/issues/589
CSIStorageCapacity | Beta | https://github.com/kubernetes/enhancements/issues/1472
GenericEphemeralVolume | Stable | https://github.com/kubernetes/enhancements/issues/1698
DaemonSetUpdateSurge | Beta | https://github.com/kubernetes/enhancements/issues/1591
CrossNamespacePodAffinity | Beta | https://github.com/kubernetes/enhancements/issues/2249

### PR DESCRIPTION
**Pull request by jainbhavya65**
This PR add alpha features in job pull-kubernetes-e2e-gce-alpha-features.
Partially fixes #23642

Signed-off-by: Bhavya Jain jainbhavya65@gmail.com

### CODE REVIEW

File: config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml [hunk 1]
```diff
@@ -405,7 +405,7 @@ presubmits:
         - --provider=gce
         - --runtime-config=api/all=true
         - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-alpha-features
-        - --test_args=--ginkgo.focus=\[Feature:(ProbeTerminationGracePeriod|APIServerTracing|StorageVersionAPI|PodPreset|StatefulSetMinReadySeconds)\]|Networking --ginkgo.skip=\[Feature:(SCTPConnectivity|Volumes|Networking-Performance)\]|IPv6|csi-hostpath-v0 --minStartupPods=8
+        - --test_args=--ginkgo.focus=\[Feature:(ProbeTerminationGracePeriod|APIServerTracing|StorageVersionAPI|PodPreset|StatefulSetMinReadySeconds|CPUManager|DynamicKubeletConfig)\]|Networking --ginkgo.skip=\[Feature:(SCTPConnectivity|Volumes|Networking-Performance)\]|IPv6|csi-hostpath-v0 --minStartupPods=8
```
Comment (endocrimes) [inline config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml:408]: `DynamicKubeletConfig` is deprecated and no longer enabled by default 😅  - Not sure if you want to still introduce it here or not

**Comment by jainbhavya65** [comment]
@cheftako Please Review This PR

**Comment by jainbhavya65** [comment]
@mmiranda96 is this PR needed?

**Comment by mmiranda96** [comment]
After the new changes, it doesn't seems to be. We can close this. Thanks for your effort anyway!
/close

**Pull request closed without merging**